modules/message-lane/src/lib.rs

```diff
@@ -164,6 +179,24 @@ decl_module! {
 		/// Deposit one of this module's events by using the default implementation.
 		fn deposit_event() = default;
 
+		/// Halt all pallet operations. Operations may be resumed using `resume_operations` call.
+		///
+		/// May only be called either by root, or by `ModuleOwner`.
+		#[weight = (0, DispatchClass::Operational)]
```

Comment:
This has at least a cost of one storage read and one storage write, no? If you intend to do benchmarks instead please add a TODO here so that we don't forget.
```suggestion
		#[weight = (T::DbWeights::get().reads_writes(1, 1), DispatchClass::Operational)]
```

Reply:
I wasn't going to do any benchmarks. The idea was that this call is kind of emergency and should fit any block. So I just left zero here - I've seen such transactions in Substrate. But probably you're right - maybe someone could spam block with these calls (even though they fail) and paying only basic tx fee.